cfg.yaml

```diff
@@ -141,6 +143,12 @@ actions:
   url:              # Mandatory. Url of Nexus IQ server
   organization-id:  # Mandatory. Organization UID like "222de33e8005408a844c12eab952c9b0"
 
+  - name: my-dependencytrack
+    type: dependencytrack
+    enable: false
+    url:                       # Mandatory. Url of Dependency Track server
```

Comment:
can we add some example?
e.g. from docker deploying - `http://localhost:8080/api`